junit5/src/main/java/cz/xtf/junit5/listeners/EventsRecorder.java

```diff
@@ -9,19 +11,44 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
+import org.junit.platform.engine.TestExecutionResult;
 import org.junit.platform.launcher.TestExecutionListener;
+import org.junit.platform.launcher.TestIdentifier;
 import org.junit.platform.launcher.TestPlan;
 
+import cz.xtf.core.config.OpenShiftConfig;
 import cz.xtf.core.openshift.OpenShift;
 import cz.xtf.core.openshift.OpenShifts;
 import io.fabric8.kubernetes.api.model.Event;
 import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
 public class EventsRecorder implements TestExecutionListener {
+    // TODO consider to remove this method as having namespace as test case should be considered as default and single namesapce as special case
```

Comment:
This TODO in scope of this MR?

Reply:
This would break current behaviour. I think we can do that when moving to XTF 2.0 one day

Reply:
We are still not even 1.0, right? :)

We do not have semantic versioning guarantees on not changing behaviour, do we? It is our own constraint limitation? Do we gather those breaking changes for next major release?

Reply:
This could be resolved?